dist_q_learning/q_estimators.py

```diff
@@ -825,17 +782,16 @@ def get_lr(self, ns=None):
 
```

Comment:
When `ns` is super low (close to 1) then this will mean we have a big learning rate, is this ok?

Reply:
I think self.lr has never been None - I am not sure if this is okay, but I think it's what we did previously for that "take the mean" thing?